src/app/(proper_react)/(redesign)/(public)/breach-details/[breachName]/BreachDetailView.module.scss

```diff
@@ -0,0 +1,199 @@
+
+  .overview {
+    width: 100%;
+    background-color: rgba($color-purple-05, 0.2);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: $spacing-lg;
+    padding: $spacing-2xl;
+
+    h2.overviewTitle,
+    .content {
+      width: $content-md;
+      max-width: 100%;
+    }
+
+    h2.overviewTitle {
+      font: $text-title-sm;
+    }
+
+    .content {
+      display: flex;
+      flex-direction: column;
+      gap: $spacing-lg;
+
+      a[href="#delayed-reporting"] {
+        color: $color-black;
+      }
+
+      h3 {
+        font: $text-title-2xs;
+      }
+
+      .compromisedDataList ul {
+        list-style-type: none;
+        display: flex;
+        flex-direction: column;
+        gap: $spacing-md;
+        margin: 0;
+        padding: $spacing-md;
+
+        li {
+          display: flex;
+          gap: $spacing-md;
+        }
+      }
+
+      small {
+        color: $color-grey-50;
+        font: $text-body-sm;
+
+        a {
+          font-weight: 700;
+          color: $color-grey-50;
+
+          &:hover {
+            text-decoration: none;
+          }
+        }
+      }
+    }
+  }
+
+  .signUp {
+    background-color: rgba($color-purple-05, 0.5);
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: $spacing-lg;
+    padding: $spacing-2xl;
+
+    .content {
+      max-width: $content-sm;
+      display: flex;
+      flex-direction: column;
+      gap: $spacing-md;
+
+      h2.signUpTitle {
+        font: $text-title-3xs;
+      }
+    }
+  }
+}
+
+.recommendations {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: $spacing-2xl;
+  padding: $spacing-2xl;
+
+  h2.recommendationTitle,
+  .lead,
+  dl {
+    width: $content-md;
+    max-width: 100%;
+  }
+
+  h2.recommendationTitle {
+    font: $text-title-xs;
+  }
+
+  dl {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-2xl;
+
+    div {
+      $iconWidth: 48px;
+
+      padding-inline-start: calc($iconWidth + $spacing-xl);
+      position: relative;
+      display: flex;
+      flex-direction: column;
+      gap: $spacing-md;
+
+      img {
+        position: absolute;
+        inset-inline-start: 0;
+        inset-block-start: 0;
+        // top: 50%;
+        // transform: translateY(-50%);
+        width: $iconWidth;
+      }
+
+      dt {
+        font: $text-title-3xs;
+      }
+
+      dd {
+        display: flex;
+        flex-direction: column;
+        gap: $spacing-md;
+        color: $color-grey-50;
+
+        .cta {
+          font-weight: 700;
+          color: $color-blue-50;
+
+          &:hover {
+            color: $color-blue-80;
+          }
+        }
+      }
+    }
+  }
+}
+
+.explainers {
+  background-color: rgba($color-purple-05, 0.2);
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: $spacing-2xl;
+  padding: $spacing-2xl;
+
+  div[id] {
```

Comment:
I don't _think_ it's particularly risky since the pattern here is a section (`.explainers`) with explanations in there that get linked to from elsewhere on the page (via the `id` attribute), but better safe than sorry :) 74839cc32